### ISSUE TITLE
DOCS Question: How to add other ssh public keys to to authorized_keys

### ISSUE THREAD

**Issue by brenzi** [flex-dapps/gantree-lib-nodejs#35]

**Comment by morelazers** [comment]
There's currently no way to do this other than manually logging into the boxes and adding those keys.

In a future update we can modify the config to accept an array of other public keys that you would like to add.

**Comment by rozifus** [comment]
Added a ticket for docs with user information on adding ssh keys

Added a ticket for multiple ssh key support at instance creation time
*This may be a bit tricky as support will vary by provider but I think it would definitely be useful where we can support it

**Comment by rozifus** [comment]
Actually on more thought we can probably add the key as part of the system provisioning after the instance is created, which should be identical across all providers

**Comment by brenzi** [comment]
I think all providers offer to supply a bunch of pubkeys for their accounts. Maybe you could just keep these as they should be provisioned with any new droplet/vps/vm?

**Comment by morelazers** [comment]
Yep @rozifus we can just take an array of keys in the config and push them into the `sshUser/.ssh/authorized_keys` file after provisioning.

**Comment by rozifus** [comment]
@brenzi, While it would be good to get the ssh keys into the nodes as early as possible I think it will be much easier to implement as a single post-provisioning step that is provider agnostic. This would also mean if we add a provider in the future that only allows for a single ssh key on creation we won't run into any issues
